ca/ca.go

```diff
@@ -219,7 +219,8 @@ func (ca *CA) Init(cfg *config.Config) (*CA, error) {
 		adminDB := auth.GetAdminDatabase()
 		if adminDB != nil {
 			acmeAdminResponder := adminAPI.NewACMEAdminResponder()
-			adminHandler := adminAPI.NewHandler(auth, adminDB, acmeDB, acmeAdminResponder)
+			policyAdminResponder := adminAPI.NewPolicyAdminResponder(auth, adminDB, acmeDB, cfg.AuthorityConfig.DeploymentType)
```

Comment:
See my previous comment.

Reply:
I'm referring to this https://github.com/smallstep/certificates/pull/788#discussion_r857965153

Reply:
Fixed in 2a7620641f74474a7a055e20c779e6c70293c32d.